src/compass/utils/h5_helpers.py

```diff
@@ -608,10 +609,14 @@ def metadata_to_h5group(parent_group, burst, cfg, save_noise_and_cal=True,
             True if (cfg.weather_model_file is not None) and \
             ('wet' in cfg.tropo_params.delay_type) else False
         tec_corr_enabled = True if cfg.tec_file is not None else False
+        who_applied_eap_correction = 'OPERA' if eap_correction_applied else 'ESA'
         par_meta_items = [
             Meta('ellipsoidal_flattening_applied',
                  bool(cfg.geocoding_params.flatten),
                  "If True, CSLC-S1 phase has been flattened with respect to a zero height ellipsoid"),
+            Meta('elevation_antenna_pattern_correction_applied',
+                 who_applied_eap_correction,
+                 "'OPERA': correction applied by s1-reader and COMPASS. 'ESA': correction was applied by ESA. 'NONE' when the correction was not applied."),
```

Comment:
```suggestion
            Meta('elevation_antenna_pattern_correction_applied',
                 who_applied_eap_correction,
                 "Antenna pattern correction. OPERA: correction applied by OPERA project; ESA: correction applied by ESA; NONE: correction not applied."),
```
Just a quick rewording. Not sure if NONE should be spelled with capital letters. I prefer `None`.

Reply:
Sounds good. In terms of `Antenna pattern` I would prefer to explicitly mention `Elevation` because there seems to be a terminology for the antenna pattern in azimuth direction. See [here](https://www.researchgate.net/figure/Azimuth-antenna-element-pattern-of-Top-Sentinel-1A-EW-mode-and-Bottom-corresponding_fig1_321505761)